src/components/App/App.js

```diff
@@ -68,6 +77,23 @@ class App extends Component {
     });
   };
 
+  openModal = (title, content, onClose) => {
+    this.setState({
+      modal: {
+        title,
+        content,
+        onClose
```

Comment:
Is it needed? Especially in `state`?

```suggestion

```

Reply:
well just try to pass it to Modal Component